ql/src/test/results/clientpositive/perf/tez/constraints/query32.q.out

```diff
@@ -115,7 +115,7 @@ Stage-0
                               Select Operator [SEL_114] (rows=285116600 width=119)
                                 Output:["_col0","_col1","_col2"]
                                 Filter Operator [FIL_112] (rows=285116600 width=119)
-                                  predicate:(cs_ext_discount_amt is not null and cs_sold_date_sk is not null)
+                                  predicate:(cs_ext_discount_amt is not null and cs_sold_date_sk is not null and cs_item_sk BETWEEN DynamicValue(RS_28_item_i_item_sk_min) AND DynamicValue(RS_28_item_i_item_sk_max) and in_bloom_filter(cs_item_sk, DynamicValue(RS_28_item_i_item_sk_bloom_filter)))
```

Comment:
Do you know if this was leading to wrong results or if the filter was ignored silently?

Reply:
That the SJ is not there will not lead to incorrect results, but can lead to a performance penalty.